### ISSUE TITLE
Activist Portal: Public event pages lack a heading element

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/app.zetkin.org#3641]
### The feature

The event names on public event pages in the activist portal are not heading elements. Here's a screenshot of one while focused in VoiceOver.

![Image](https://github.com/user-attachments/assets/f29ef96e-216c-4404-ae7f-923befbc3605)

### The bug

The two most popular screen readers in the world are JAWS and NVDA, and in both of them, the `h` key is used to focus the next heading on the page. Pressing `h` to flick through the headings of a page is such an important part of the workflow of a visually impaired user learning their way around an unfamiliar new site that there are two WCAG criteria that touch on the importance of providing headings on a page.

1. [1.3.1. Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
2. [2.4.6: Headings and Labels](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels.html)

Using `h1` - `h6` elements for headings is how to comply with these success criteria when it comes to headings. Currently we have a `<p>` tag on this element.

## Steps to reproduce

1. Go to http://localhost:3000/o/1/events/535
2. Fire up VoiceOver
3. Focus `Test event 2 with looooong link` like in the screenshot

### Expected Behaviour

VoiceOver should announce

> Heading level 1, Test event 2 with looooong link

### Actual Behaviour

It's just plain old

> Test event 2 with looooong link
